Analysis/Utkscan/core/include/HighResTimingData.hpp

```diff
@@ -41,7 +41,8 @@ class HighResTimingData : public ChanEvent {
         return(false);
     }
 
-    /** \return The current value of aveBaseline_ */
+    /** \return The current value of aveBaseline_ .
+      This value is calculated during the fitting procedure as a straight sum, from the trace. */
```

Comment:
The note about this should come before the return. Also, the baseline average is not a "straight sum", this may mislead users to think that we are simply calculating a QDC. Maybe update the copy to read

> This baseline is calculated from the trace by averaging the points preceding the waveform.